bc_obps/service/tests/test_operation_service_v2.py

```diff
@@ -708,136 +830,105 @@ def test_update_operation_with_operation_representatives_with_address():
             operation_representatives=[contact.id for contact in contacts],
         )
 
-        operation = OperationServiceV2.create_or_update_operation_v2(
+        operation = OperationServiceV2.update_operation(
             approved_user_operator.user.user_guid,
             payload,
             existing_operation.id,
         )
         operation.refresh_from_db()
         assert Operation.objects.count() == 1
         assert operation.contacts.count() == 3
-        assert operation.updated_at is not None
-
 
-class TestOperationServiceV2UpdateOperation:
-    def test_update_operation_fails_if_operation_not_registered(self):
```

Comment:
We no longer need the operation to be registered before updating since the refactor